brokers/vscode/broker_test.go

```diff
@@ -164,7 +194,7 @@ func TestProcessPluginNoAttributes(t *testing.T) {
 
 	err := m.b.processPlugin(meta)
 
-	assert.EqualError(t, err, "VS Code extension field 'extension' is missing in description of plugin tid:tv")
+	assert.EqualError(t, err, "'attributes' are not found in VS Code extension description of the plugin tid:tv")
```

Comment:
There are 2 cases:
- no attributes
- no extension attribute 
Are they both covered?

Reply:
* TestProcessPluginWithExtension - only `extension` attribute exists
* TestProcessPluginWithUrl - only `url` attribute exists
* TestProcessPluginNoExtensionNoUrl - `extension` and `url` are missed

Reply:
ok